app/Console/Kernel.php

```diff
@@ -4,6 +4,7 @@
 
 use Illuminate\Console\Scheduling\Schedule;
 use Illuminate\Foundation\Console\Kernel as ConsoleKernel;
+use Ramsey\Uuid\Generator\CombGenerator;
```

Comment:
ooooh what does the comb generator do?

Reply:
I dunno! IDE flags it a unused. I may have got helpfully added by a typo?

Reply:
lol